### PR TITLE
Feature/sr 13 implement products page

### PR DESCRIPTION
**Pull request by masterHAWK99**

### CODE REVIEW

File: src/components/filters/FiltersButton.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <div className="filters-wrapper">
+      <button
+        className="btn filters-button"
+        onClick={() => handleSetCollapsed(true)}
+      >
+        <img src={sortIcon} alt="" />
+        All filters
+        <img src={chevronDownIcon} alt="" />
+      </button>
```
Comment (GalacticOctopus) [inline src/components/filters/FiltersButton.tsx:26]: Specify alt attributes

File: src/components/products/SortButton.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  {
+    method: (a: Scooter, b: Scooter) => a.amount - b.amount,
+    name: "Best choice",
+  },
+  {
+    method: (a: Scooter, b: Scooter) => b.amount - a.amount,
+    name: "Popular",
+  },
+];
+
```
Comment (mzajaczkowsky) [inline src/components/products/SortButton.tsx:27]: Could you please add comments to enable the identification of sorting functions?

File: src/scss/base/_typography.scss [hunk 3]
```diff
@@ -46,10 +46,18 @@
   line-height: 140%;
 }
 
-.text-700 {
+.font-weight-700 {
   font-weight: 700;
 }
```
Comment (mzajaczkowsky) [inline src/scss/base/_typography.scss:51]: Please stick to one naming convention.

**Review by mzajaczkowsky — DISMISSED** [review]

File: src/components/filters/FiltersButton.tsx (hunk 1 above)
Comment (masterHAWK99) [reply src/components/filters/FiltersButton.tsx:26]: Those icons are only decorative, "All filters" determine what this button does.

**Comment by GalacticOctopus** [comment]
Filtering and sorting works, we still need to add better rwd, but at this point we can merge. Well done.

**Review by GalacticOctopus — APPROVED** [review]
Filtering and sorting works, we still need to add better rwd, but at this point we can merge. Well done.

**Pull request merged**